index.md

```diff
@@ -20,6 +20,5 @@ about:
 <br>
 <br>
 
-R-multiverse is a general-purpose R package repository system with
-[transparent governance](policies.md) and open source infrastructure.
-Visit the [overview page](overview.qmd) to learn more.
+R-multiverse is a new initiative to establish a democratic, multi-stakeholder, and trustworthy R packages repository that runs on the open-source [r-universe](https://r-universe.dev) infrastructure. We strive for [transparent governance](policies.md), high automatation, and quarterly production releases. Visit the [overview page](overview.qmd) page to learn more.
```

Comment:
```suggestion
R-multiverse is a new initiative to establish a trustworthy, community-curated R packages repository that runs on the open-source [r-universe](https://r-universe.dev) infrastructure. We strive for [transparent governance](policies.md), high automatation, and quarterly production releases. Visit the [overview page](overview.qmd) page to learn more.
```

Reply:
I preferred your original snappier version 😄

Reply:
I thought it would be good to somehow emphasize the multi stands for multiple organizations or stakeholders are involved (to contrast with e.g. cran). But you prefer community-curated?

Reply:
I do like to have 'community' in there somewhere - that lends it legitimacy as well as just being 'nice'.